Requirements/Requirements_And_Goals.md

```diff
@@ -0,0 +1,73 @@
+# Requirements and Goals
+
+## Mission
+
+> The Hey, Blue! initiative facilitates moments of meaningful connection between police officers and members of their community by offering intentional opportunities for individuals to meet and share positive experience.
+
+## Goal
+
+To provide a cost effective platform which helps the civilians connect with police officers for a positive interaction experience at a scale in a secure way. This platform can be hosted on both web and mobile based. 
+
+## Actors
+- Citiens
+- Police officers
+- Charities
+- Retail Organisations
+
+## Functional Requirements
+
+### Hard Requirements 
+ 1. End-Use Ease of use (HR1)
+ 2. Location Tracking (HR2)
+ 3. Connecting with other users (HR3)
+ 4. Uploading to social media (HR4)
+ 5. Tracking engagements
+ 6. Ability to opt in to and out of push Notifications
+ 7. Users can only have one connection per officer per 24hour period
+ 8. Community members can find officers and not the other way around
```

Comment:
Be consistent, either use `civilian` or ` community member` everywhere

Reply:
Thats the language coming from the requirement.

Reply:
Ok, still better to choose one and be consistent ourselves